nav_msgs/Odometry waypoint
---
bool success
string message